stylix/palette.nix

```diff
@@ -124,5 +126,6 @@ in {
     # This attrset can be used like a function too, see
     # https://github.com/SenchoPens/base16.nix#mktheme
     lib.stylix.colors = (base16.mkSchemeAttrs cfg.base16Scheme).override override;
+    lib.stylix.scheme = base16.mkSchemeAttrs cfg.base16Scheme;
```

Comment:
I feel like having both of these exposed could cause some confusion, as only `colors` is meant to be used for theming. I see why it's necessary so I'll merge this as is for the time being.

Reply:
Yeah maybe instead of using `lib.stylix.scheme` I could have used an option, so that I could mark it as `internal` and `readOnly`, that way it wouldn't show on documentation.